### PR TITLE
DONT'T MERGE IN WORK!

### PR DESCRIPTION
**Pull request by xISRAPILx**
### Purpose and Effect of This Pull Request
Add support new version!

### Testing
no

### CODE REVIEW

File: src/pocketmine/network/protocol/Info.php [hunk 1]
```diff
@@ -25,51 +25,51 @@
...
 	 * Actual Minecraft: PE protocol version
 	 */
 
-	const CURRENT_PROTOCOL = 113;
-	const ACCEPTED_PROTOCOLS = [110, 111, 112, 113];
-	const MINECRAFT_VERSION = ["v1.1.0", "v1.1.1", "v1.1.2", "v1.1.3", "v1.1.4"];
-	const MINECRAFT_VERSION_NETWORK = "1.1.4";
+	const CURRENT_PROTOCOL = 121;
+	const ACCEPTED_PROTOCOLS = [120, 121];
+	const MINECRAFT_VERSION = ["v1.2.0.1", "v1.2.0.2"];
```
Comment (Leo3418) [inline src/pocketmine/network/protocol/Info.php]: This is wrong. The versions are v1.2.0.2 and v1.2.0.7.

**Comment by SuperXingKong** [comment]
I am wondering how to find these...

**Comment by xISRAPILx** [comment]
what? @SuperXingKong

**Comment by xISRAPILx** [comment]
https://github.com/NiclasOlofsson/MiNET/blob/9f4f62721717d2f550a7145d48af88c46e5b7344/src/MiNET/MiNET/Net/MCPE%20Protocol%20Documentation.md @SuperXingKong

**Comment by SuperXingKong** [comment]
@xISRAPILx Thabks!

**Pull request closed without merging**